pkg/fakes/cluster.go

```diff
@@ -1,524 +0,0 @@
-// Code generated by MockGen. DO NOT EDIT.
-// Source: github.com/rancher/webhook/pkg/generated/controllers/management.cattle.io/v3 (interfaces: ClusterController,ClusterClient,ClusterCache)
-
-// Package fakes is a generated GoMock package.
-package fakes
```

Comment:
You mind elaborating on why this file was deleted?

Reply:
It was not being used, so I didn't add it to the automatic generation.